### PR TITLE
Added combsort,egyptian fraction and minimum coins in C

### PR DESCRIPTION
**Pull request by YashovardhanSiramdas**
**Changes:**
Added 3 new files in C:-
1)Comb Sort
2)Egyptian Fraction
3)Minimum Coins

### CODE REVIEW

File: code/greedy_algorithms/src/minimum_coins/minimum_coins.c [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+
+int 
+main()
+{
+	printf("Enter Number of Denominations \n");
+	int n;
+	scanf("%d", &n);
+	int denominations[n], frequency[n];
+	for(int i = 0; i < n; i++)
```
Comment (arnavb) [inline code/greedy_algorithms/src/minimum_coins/minimum_coins.c]: You may want to move i outside the for loop for compatibility with older C versions.

File: code/greedy_algorithms/src/minimum_coins/minimum_coins.c [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	printf("Enter Bill \n");
+	int bill;
+	scanf("%d", &bill);
+	int i = 0;
+	while (bill > 0 && i < n) {
+		frequency[i] = bill / denominations[i];
+		bill = bill - (frequency[i] * denominations[i]);
+		i++;
+	}
+	if (bill == 0 ) {
```
Comment (arnavb) [inline code/greedy_algorithms/src/minimum_coins/minimum_coins.c]: No need for braces.

File: code/sorting/src/comb_sort/comb_sort.c [hunk 3]
```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+
+int
+updateGap(int gap)
+{
+    gap = (gap * 10) / 13;
+    if (gap < 1)
```
Comment (arnavb) [inline code/sorting/src/comb_sort/comb_sort.c]:
```C
return gap < 1 ? 1 : gap;
```

**Comment by YashovardhanSiramdas** [comment]
@arnavb @AdiChat I've made changes mentioned accordingly. Can anyone help me with the new Travis check? It shows "All checks have failed".Is this something related to style guide?

**Comment by YashovardhanSiramdas** [comment]
Travis CI checks that the code passes all the test cases. Is it fully deployed for cosmos right now? Do we have test cases to run for each program ?

**Comment by arnavb** [comment]
@YashovardhanSiramdas The current CI only is configured for Ruby, so you might as well just ignore it. Your code is fine.

**Comment by AdiChat** [comment]
Looks great 😍 

We are testing Travis CI and other tools/ services over Cosmos. You may ignore them. 
Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**